api/serializers.py

```diff
@@ -38,3 +38,13 @@ class Meta:  #pylint: disable=too-few-public-methods
         # Return optional Profile fields.
         model = Profile
         fields = ('id', 'facebook_user', 'phone', 'profile',)
+
+
+
+class AddLoanSerializer(serializers.ModelSerializer):
+    #Adding of load detail
+    class Meta:
+        #return option Loan
+            model=add_loan_record
+            fields=('loan','user_id','created','amount','interest_rate','paid','borrower','description','balance_to_pay')
```

Comment:
Most of your fields were set to blank True, so you might get errors here because they were never parsed

Reply:
not all except for balance_to_pay,  description 
which i have remove the  descrition(blank)
`class add_loan_record(models.Model):
    loan=models.CharField(max_length=200, null=True)
    user_id=models.ForeignKey(User,on_delete=models.CASCADE)
    due_date=models.DateField()
    created=models.DateField(auto_now_add=True)
    amount=models.IntegerField()
    interest_rate=models.DecimalField(max_digits=19, decimal_places=10)
    paid=models.BooleanField(default=False)
    borrower=models.BooleanField(default=True)
    description=models.TextField()
    balance_to_pay=models.CharField(max_length=200,blank=True)
`